pkg/controller/admissionchecks/multikueue/workload_test.go

```diff
@@ -1050,7 +1050,7 @@ func TestWlReconcile(t *testing.T) {
 			}
 
 			helper, _ := newMultiKueueStoreHelper(managerClient)
-			reconciler := newWlReconciler(managerClient, helper, cRec, defaultOrigin, defaultWorkerLostTimeout)
+			reconciler := newWlReconciler(managerClient, helper, cRec, defaultOrigin, defaultWorkerLostTimeout, time.Second)
```

Comment:
I would be ok not to parametrize it. but if you do, then let's use 100ms to get (slightly) better runtimes.

Reply:
The value is not relevant in this test.